index.html

```diff
@@ -0,0 +1,1224 @@
+            </div>
+          </section>
+
+          <section id="acl-resources" inlist="" rel="schema:hasPart" resource="#acl-resources">
+            <h2 property="schema:name">ACL Resources</h2>
+            <div datatype="rdf:HTML" property="schema:description">
+              <p>This section describes ACL resource <a href="#acl-resource-discovery" rel="cito:discusses">discovery</a> and <a href="#acl-resource-representation" rel="cito:discusses">representation</a>.</p>
+
+              <section id="acl-resource-discovery" inlist="" rel="schema:hasPart" resource="#acl-resource-discovery">
+                <h3 property="schema:name">ACL Resource Discovery</h3>
+                <div datatype="rdf:HTML" property="schema:description">
+                  <p id="req-server-link-acl">When a server wants to enable applications to discover <a href="#authorization">Authorizations</a> associated with a given <a href="#resource">resource</a>, the server MUST advertise the <a href="#acl-resource">ACL resource</a> that is associated with a resource by responding to an HTTP request including a <code>Link</code> header with the <code>rel</code> value of <code>acl</code> (<cite><a href="#acl-link-relation" rel="rdfs:seeAlso">acl Link Relation</a></cite>) and the ACL resource as link target.</p>
+
+                  <p>ACL Resource Discovery is used towards determining the <cite><a href="#effective-acl-resource" rel="rdfs:seeAlso">Effective ACL Resource</a></cite> of a resource.</p>
+
+                  <div class="issue" id="link-relation-type" rel="schema:hasPart" resource="#link-relation-type">
+                    <h4 property="schema:name"><span>Issue</span>: Link Relation Type</h4>
+                    <div datatype="rdf:HTML" property="schema:description">
+                      <p>The possibility of using URIs as relation types interchangeably or as alternate to the <cite><a href="#acl-link-relation" rel="rdfs:seeAlso">acl Link Relation</a></cite> type is under consideration, e.g., <code>https://www.w3.org/ns/iana/link-relations/relation#acl</code> or <code>http://www.w3.org/ns/auth/acl#accessControl</code>: <a href="https://github.com/solid/web-access-control-spec/issues/21#issuecomment-769137284" rel="cito:citesAsRelated">issues/21</a>.</p>
+                    </div>
+                  </div>
+
+                  <p id="req-server-resource-acl-max">Servers MUST NOT directly associate more than one ACL resource to a resource.</p>
```

Comment:
IIRC, the text is derived from Solid Protocol on ACL auxiliary resources. Besides that, I don't recall min/max in prior WAC material. I haven't seen any implementation advertising more than one relation either.

Multiple associations would potentially require the server to advertise all in Link header(s). May be complicating both servers and applications in the end. I suppose there can be multiple ACL resources associated but I'm not sure if there is a strong need. We can revisit this if you like though in an issue?

Reply:
The main advantage of allowing the link to multiple effective ACL would probably be in reuse. But then, I can see how this clashes with the current inheritance mechanism.

I'm starting to think that basing inheritance on a more generic mechanism for reuse would be potentially a good way to go.

I'm fine with the idea of one effective ACL per resource since it's what we're talking about here.

Reply:
The main advantage of allowing the link to multiple effective ACL would probably be in reuse. But then, I can see how this clashes with the current inheritance mechanism.

I'm starting to think that basing inheritance on a more generic mechanism for reuse would be potentially a good way to go.

I'm fine with the idea of one effective ACL per resource since it's what we're talking about here.

Reply:
I can see the advantage of multiple ACLs per resource (perhaps one that is the main ACR, and then others for example pointing to say the owner document, that always gives the owner access. 

But right now as I am implementing tests for WAC+:imports I realise that a client who would want to edit the principal one for a resource, will want to know what to edit, and having multiple links to different acls would make that job quite complicated. I think one can find other ways of getting the same behavior.  So +1 for an issue on this.